app/views/posts/index.html.erb

```diff
@@ -1 +1,40 @@
-<h1>Here is a list of all posts</h1>
\ No newline at end of file
+<%= stylesheet_link_tag ('home') %>
+
+<section class="post-section">
+  <div>
+  <div class="user-main">
+    <div class="photo">
+      photo
+    </div>
+    <div class="name">
+      <ul>
+        <li>
+          Username: <%= @user.name %>
+        </li>
+      </ul>
+        <p class="user-activity">
+          <%= link_to "view post", user_path(id: @user.id) %>
+        </p>
+    </div>
+  </div>
+
+  <div class="post-list">
+    <% @posts.each_with_index do |post, index| %>
+    <div class="posts-body">
+      <h2>Post #<%= index%></h2>
+      <p><%= post.text %></p>
+      <div class="post-activity">
+        Comments: <%= post.comment_counter %>,
+        Likes: <%= post.likes_counter %>
+      </div>
+    </div>
+    <div class="comments">
+      <h3>Username: <%= @user.name %></h3>
+      <h3>Comments:  <%= post.comment_counter %></h3>
+    </div>
```

Comment:
- [ ] Kindly consider using `Post.recent_comments` to render the recent 5 comments of each post in the user post index page (This page)